tests/pageObjects/AddProduct.js

```diff
@@ -0,0 +1,31 @@
+class AddProduct{
+    constructor(){
+        this.accountSelector ="//button[@id='account-button']";
+        this.adminPanelSelector = "//a[contains(text(), 'Admin Panel')]";
+        this.sidebarProductSelector = "//a[@href = '#sidebar-products']";
```

Comment:
please look other lines also
```suggestion
        this.sidebarProductSelector = "//a[@href='#sidebar-products']";
```